flux/stdlib/influxdata/influxdb/rules.patch

```diff
@@ -0,0 +1,48 @@
+diff --git a/flux/stdlib/influxdata/influxdb/rules.go b/flux/stdlib/influxdata/influxdb/rules.go
```

Comment:
need to git rm this?

Reply:
That was intentionally kept. I was thinking about maybe doing it for everything that I modified so I don't have to do this again? I wasn't sure if it was a good idea or not.